cdk/projen-common.js

```diff
@@ -18,4 +18,5 @@ exports.options = {
     license: 'MIT',
     licensed: true,
     gitignore: ['/docs'],
+    defaultReleaseBranch: 'master',
```

Comment:
Is this a new required property from projen? I don't think we would need it since we don't use their CI config

Reply:
i updated the projen version and it throws error without it:
Error: "defaultReleaseBranch" is temporarily a required option while we migrate its default value from "master" to "main"

Reply:
Ah, well, I guess there's nothing we can do about that then